app/Http/Controllers/UserProjectsController.php

```diff
@@ -46,9 +45,11 @@ public function showContracted(): Response|View
         $user = Auth::user();
 
         if (in_array($user->context, ['individual', 'organization'])) {
-            if ($user->individual && $user->individual->isParticipant() && ($user->individual->isConsultant() || $user->individual->isConnector())) {
+            $userContext = $user->individual ?? $user->organization;
+            if ($userContext && ($userContext->isConsultant() || $userContext->isConnector() || $userContext->inProgressContractedProjects()->count())) {
                 return view('projects.my-projects', [
                     'user' => $user,
+                    'projectable' => $user->organization ?? null,
```

Comment:
The nullish coalescence is unnecessary here because it will only trigger if the left side is already `null`.
```suggestion
                    'projectable' => $user->organization,
```